zerver/tests/test_soft_deactivation.py

```diff
@@ -770,6 +772,17 @@ def assert_stream_message_not_sent_to_idle_user(
         )
         assert_stream_message_not_sent_to_idle_user("Test @**bogus** mention")
 
+        # Test UserMessage row is created while user is deactivated if
+        # there is a topic wildcard mention i.e. @topic
+        do_soft_activate_users([long_term_idle_user])
+        self.send_stream_message(long_term_idle_user, stream_name, "Hi", topic_name)
+        topic_participant_user_ids = {long_term_idle_user.id}
+
+        assert_stream_message_sent_to_idle_user(
+            "Test @**topic** mention", topic_participant_user_ids=topic_participant_user_ids
+        )
+        do_soft_deactivate_users([long_term_idle_user])
```

Comment:
Should we be first deactivating, then testing here? We need to check that message is created while the user is soft-deactivated, correct?